docs/getting_started.md

```diff
@@ -84,7 +84,77 @@ work for Windows? (If somebody could verify this, that would be great!)
 
 ### OSX
 
-TODO
+NOTE: These commands were tested with Intel MacOS 13.0, they may or may not not work 
+with older versions.
+
+#### Installing the Haskell toolchain
+
+Run the following commands (see https://www.haskell.org/ghcup/) to install `ghcup`, `ghc` and `cabal`. 
+
+
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
+$ ghcup tui
+# In the terminal UI, select GHC9.0.2, Haskell language server 1.8 and Cabal 3.6.
+# Important: both install + set them! 
+```
+![](https://www.haskell.org/ghcup/ghcup.gif)
+
+`hpack` and `hspec-discover` are also installed but they are only needed when working on the compiler.
+
+```bash
+$ cabal install hpack
+$ cabal install hspec-discover
+```
+
+Verify you installed the correct versions by running the commands below, and
+comparing them against the versions mentioned in the previous command:
+
+```bash
+$ ghc --version
+$ haskell-language-server-wrapper --version
+$ cabal --version
+```
+
+#### Installing Souffle
+
+Run the following commands to download and build Souffle from source (instructions taken from https://souffle-lang.github.io/build#mac-os-x-build):
```

Comment:
Good one, if it ever becomes outdated, we can directly look up the newer instructions! :trophy: